DEVELOPMENT.md

```diff
@@ -81,13 +81,17 @@ docker compose exec es_workspace bin/migrate
 While developing you may use the watch mode to rebuild the system on changes
 
 ```bash
-# Run a front-end build and update on change
+# Run a front-end build and update on change. Note that this doesn't apply to changed translation files – see above on how to update them manually
```

Comment:
"Frontend build" should have no connotation of "doing translations" (as the don't have to be build; using `.mo` files is only recommended for prod; dev works without problems when directly parsing the `.po`.

So something like
```suggestion
# Run a front-end build and update on change (JS & CSS, no translations)
```

Reply:
As someone who is not very familiar with frontend development and just wanted to see if I could tweak some things we were struggling with, all of these distinctions were not obvious to me (I changed some wording in a .po file to see if my setup was working, nothing changed -> I looked for the error in my setup). The wording I added here was an attempt to help other newbies like me – I'm very open to finding shorter ways to express this, but `(JS & CSS, no translations)` would not have helped me made the connection. 

How about something like...

```suggestion
# Run a front-end build and update every tome a JS or CSS file is changed (not translation files!)
```
?

Additionally, your comment makes me question whether I messed up my setup somehow because I still have to do the .po->.mo updates manually in order for changes in .po files to show up in my dev instance. I'll investigate and might come back with an additional PR :D

Reply:
The `.mo` files are binary files that are easier to parse and thuis give a small performance boost on prod systems but the angelsytem is totally useable when they are not present, it uses the text `.po` files automatically.

So yees, you should delete your `.mo` files for the dev env ;)